libraries/RC_Channel/RC_Channel.h

```diff
@@ -418,6 +418,7 @@ class RC_Channels {
     // mode switch handling
     void reset_mode_switch();
     virtual void read_mode_switch();
+    bool check_flight_mode_channel_conflict(uint8_t &mode);
```

Comment:
```suggestion
    bool check_flight_mode_channel_conflict(uint8_t &mode);
```

Should be `const`

Reply:
I created another commit to add const.
please review it.